src/commands/add.ts

```diff
@@ -15,7 +15,12 @@ function createHookFile(file: string, cmd: string) {
     throw new Error(`${file} already exists`)
   }
 
-  const data = ['#!/bin/sh', '. "$(dirname $0)/_/husky.sh"', '', cmd].join('\n')
+  const data = [
```

Comment:
```suggestion
    const data = ['#!/bin/sh', '. "$(dirname "$0")/_/husky.sh"', '', cmd].join('\n')
```

Reply:
Done, except for the additional indentation (?) :)